### PR TITLE
Pub score fix

### PR DESCRIPTION
**Pull request by mannprerak2**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -3,9 +3,9 @@
 # BSD-style license that can be found in the LICENSE file.
 
 name: ffigen
-version: 0.1.0
+version: 0.1.1
 homepage: https://github.com/dart-lang/ffigen
-description: Experimental generator for FFI bindings.
+description: Experimental generator for FFI bindings, uses LibClang to parse C/C++ header files.
```
Comment (dcharkes) [inline pubspec.yaml:8]: uses -> using

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# 0.1.1
+- Fix pub score.
```
Comment (dcharkes) [inline CHANGELOG.md:2]: - Address pub score: follow dart File conventions, provide documentation, and pass static analysis.

File: lib/ffigen.dart [hunk 3]
```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2020, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// A bindings generator for dart.
+///
+/// See complete usage at - https://pub.dev/packages/ffigen.
+library ffigen;
+
+export 'src/code_generator.dart';
```
Comment (dcharkes) [inline lib/ffigen.dart:10]: You might want to be particular about what you allow to export from these three files. Once people start using this, you cannot change it anymore.

Comment (mannprerak2) [reply lib/ffigen.dart]: Ah, right.
Updated to expose just the things required to build your own `bin/ffigen.dart` script. If the user wants even more flexibility they can always import from `lib/src`.

**Pull request merged**